types/theming/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for theming 1.3
+// Project: https://github.com/iamstarkov/theming#readme
+// Definitions by: Sebastian Silbermann <https://github.com/eps1lon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+import * as React from "react";
+
+export type Channel = string;
+export type Theme = object | ((outerTheme: object) => object);
```

Comment:
Technically this is just equivalent to `object`

Reply:
At a glance, I'd replace this with `Record<string, any> | ((outerTheme: object) => object)`.

Reply:
I was wondering how I should express a plain object. It's basically a plain object or a function that is given a plain object and returns one. I guess Record is a better option than object since that excludes arrays and functions?

Reply:
Mmm, maybe it's not any better. Maybe this is fine on its own.